### PR TITLE
Panel colour and wing animation improvements

### PR DESCRIPTION
**Pull request by sryu1**
* Fix aileron and horizontal stabilizer animation for roll movements
* Change some panel colours to green

Cockpit will definitely need a new 3d model, can't really do anything with the cockpit until it's fixed.

### CODE REVIEW

**Comment by SammySkycrafts** [comment]
@sryu1 Please check that last commit. You are calling something from an external aircraft that is unused.

**Comment by sryu1** [comment]
I'm assuming you're talking about the `/accelerations/pilot-gdamped`... I did test it and it does exist in properties...
image

File: Nasal/ext_stores.nas
Comment (SammySkycrafts) [inline Nasal/ext_stores.nas:1]: This should not be here. It would more than likely not have done anything for you during testing wither.

**Review by SammySkycrafts — CHANGES_REQUESTED** [review]
@sryu1 Please take a look at the changes needed below. That dialog likely would not have worked for you during testing either, so It wouldn't be needed.

File: Nasal/ext_stores.nas
Comment (sryu1) [reply Nasal/ext_stores.nas:1]: Oh, that one was raising errors in the console so I just commented it out. I'll remove that line.

**Review by SammySkycrafts — APPROVED** [review]
Thanks!

**Pull request merged**